.gitignore

```diff
@@ -1,3 +1,4 @@
 *.gem
 ignore_list.txt
-coverage/
\ No newline at end of file
+coverage/
+Gemfile.lock
```

Comment:
Can we sort the contents of this file please?

Reply:
Sure!